### PR TITLE
BNH-192 - landlord profile restyling

### PR DESCRIPTION
**Pull request by cec200wire**
Added modal with a brief explanation of what the 'Invite link' functionality is (this caused confusion during UAT).
Restyled the buttons on the page to make them equally-sized and more distinguished from each other.
Fixed a minor formatting bug with the profile picture overlapping with the name/profile type on smaller screens.

### CODE REVIEW

**Comment by emilypatsko** [comment]
Please could you add screenshots/similar?

**Comment by cec200wire** [comment]
Screenshots:
image
image

File: web/Views/Landlord/Profile.cshtml [hunk 1]
```diff
@@ -186,4 +202,21 @@ else
...
+<!-- InviteModal -->
+<div class="modal fade" id="inviteModal" tabindex="-1" aria-labelledby="inviteModalLabel" aria-hidden="true">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title" id="approveModalLabel">Landlord invite links</h5>
+                <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+            </div>
+            <div class="modal-body">
+                An invite link allows you to assign a landlord profile to a new user of the Bricks & Hearts site.
```
Comment (emilypatsko) [inline web/Views/Landlord/Profile.cshtml:216]: Ampersands (and other characters) can cause issues - might want to look at this https://mrcoles.com/blog/how-use-amersands-html-encode/

**Review by emilypatsko — CHANGES_REQUESTED** [review]

File: web/Views/Landlord/Profile.cshtml (hunk 1 above)
Comment (cec200wire) [reply web/Views/Landlord/Profile.cshtml:216]: Should I change & to &amp; in the other places it appears as well? There are definitely at least a couple of other places that 'Bricks & Hearts' is written on the site, and I can find and replace them all if that will avoid bugs.

**Review by emilypatsko — APPROVED** [review]

**Pull request merged**